nemo_skills/evaluation/metrics/utils.py

```diff
@@ -43,3 +42,32 @@ def is_correct_judgement(judgement):
         return False  # improper judgement format, so have to judge as false
     verdict = judgement.split('Judgement:')[-1].strip()
     return verdict.lower() == 'yes'
+
+
+def get_metrics(metric_type: str):
+
+    from nemo_skills.evaluation.metrics.answer_judgement_metrics import AnswerJudgementMetrics
+    from nemo_skills.evaluation.metrics.arena_metrics import ArenaMetrics
+    from nemo_skills.evaluation.metrics.code_metrics import CodeMetrics
+    from nemo_skills.evaluation.metrics.if_metrics import IFMetrics
+    from nemo_skills.evaluation.metrics.lean4_metrics import Lean4Metrics
+    from nemo_skills.evaluation.metrics.math_metrics import MathMetrics
+    from nemo_skills.evaluation.metrics.mtbench_metrics import MtBenchMetrics
+
+
+    METRICS_MAP = {
+        "metric-math": MathMetrics(),
```

Comment:
I think metric- is redundant here, let's just call it `math`, `lean4-proof`, etc. We already call it through the get_metrics, so it's clear that this is about metrics.

Reply:
I put metric, just to avoid the same name for eval_type and metric

Reply:
I think it's fine that name for eval and metric is the same as most of the time that should be the case (e.g. both eval type and metric type is "math" makes sense to me)

Reply:
done